src/main.js

```diff
@@ -0,0 +1,36 @@
+// Load in the file-system module from the node standard-library
+// Whenever anybody talks about a "standard library", they're just talking'
+// about a library of code that the creators of the language give you
+// to make the code more useful
+// https://nodejs.org/dist/latest-v16.x/docs/api/fs.html
+const fs = require('fs');
+
+/**
```

Comment:
This format for commenting javascript code is known as [JSDoc](https://jsdoc.app/). It's a common standard for documenting JS code and many text-editors can use the comments to give readers information.

Reply:
JSDoc is new to me, but seems relatively straight forward in practice.